### PR TITLE
Branch code review

### PR DESCRIPTION
**Pull request by mutesihope**
This is Html and CSS files for code review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    </header>
+
+    <main>
+      <section id="welcome-section" class="welcome">
+        <h1>Hey I am Mimic</h1>
+        <p>a web developer</p>
+      </section>
+
+      <section id="projects" class="flow">
+        <h1>These are some of my projects</h1>
```
Comment (mutesihope) [inline index.html:30]: This line looks beautiful but it could be better if it's  h2 for it not to be the main heading. also for semantics reasons

**Review by mutesihope — COMMENTED** [review]
The code looks beautiful but you may change about your html file on h1 below for the sake of semantics

**Review by mutesihope — COMMENTED** [review]
This is general comment so please try to add a descriptive readme file.

**Review by Olamarx — APPROVED** [review]
Hi @mutesihope,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good use of Github review feature ✔️
- Grammatically correct review ✅ 
- Polite choice of words ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**